packages/driver/src/cy/actionability.coffee

```diff
@@ -186,6 +186,7 @@ ensureElIsNotCovered = (cy, win, $el, fromViewport, options, log, onScroll) ->
   return $elAtCoords
 
 getCoordinatesForEl = (cy, $el, options) ->
+  debugger
```

Comment:
Y'all need some linters running on precommit hooks, debugger should never been committed. 👀

Reply:
its a WIP, so i use `--no-verify`

Reply:
I don't think we should bother reviewing WIP PR's, isn't that the point of WIP? :smile:

Reply:
Agree with @Bkucera 	if we indicate a WIP on a PR it should be ignored and not be reviewed.